docs/Architecture Decision Record/016-dependabot-dependency-management.md

```diff
@@ -23,6 +23,9 @@ To maintain our gitflow:
 ![disable-dependabot-security-updates](https://user-images.githubusercontent.com/22626085/118340020-8b744f80-b4e8-11eb-8bb1-eb851f074627.png)
 * We also need to specify a target_branch in the dependabot configuration file to prevent automated version-related PR updates from opening in the HHS repo.
 
+*: An unforeseen consequence of this switchover was that Synk was missing out on version updates that Dependabot is now flagging. We have accummulated a significant but unknown size backlog as dependabot is limited to 10 PRs at a time. As a response, we are proposing a mass-merge in #1023 instead of individual PRs per version update due to the laborious nature of this stream of updates. We will manually create a unique branch and modify the configuration file to have dependabot scan for updates on this new target branch without the numeric limit for PRs opened. Once all are merged, we will follow our gitflow process and open a PR to 'tdp-raft-main' branch and have raft-review and so on. For merging those PRs, the github-native dependabot does not have auto-merge capabilities so these would have to be done manually or we'd have to add a ![third-party bot](https://github.com/ahmadnassri/action-dependabot-auto-merge) for this limited use-case. The above outlined process is an as-needed process to clear the backlog -- once the backlogged is cleared, we will resume the previous process of one PR per update.
```

Comment:
```suggestion
*: An unforeseen consequence of this switchover was that Synk was missing out on version updates that Dependabot is now flagging. We have accummulated a significant but unknown size backlog as dependabot is limited to 10 PRs at a time. As a response, we are proposing a mass-merge in #1023 instead of individual PRs per version update due to the laborious nature of this stream of updates. We will manually create a unique branch and modify the configuration file to have dependabot scan for updates on this new target branch without the numeric limit for PRs opened. Once all are merged, we will follow our gitflow process and open a PR to 'tdp-raft-main' branch and have raft-review and so on. For merging those PRs, the github-native dependabot does not have auto-merge capabilities so these would have to be done manually or we'd have to add a ![third-party bot](https://github.com/ahmadnassri/action-dependabot-auto-merge) for this limited use-case. The above outlined process is an as-needed process to clear the backlog -- once the backlog is cleared, we will resume the previous process of one PR per update.
```